docs/chapters/03-3.md

```diff
@@ -107,8 +112,7 @@ In this notebook, we train a multilayer perceptron (FC network) and a convolutio
 
 
 <img src="Figure 8 Instances from the Original MNIST Dataset.png" width="350px" /><br>
```

Comment:
`<center>` is missing.

Reply:
Fig. 8 is centered. I see it in my branch